cmd/dtab.go

```diff
@@ -108,12 +108,11 @@ var (
 	}
 
 	dtabUpdateCmd = &cobra.Command{
-		Use:     "update [name] [file]",
+		Use:     "update [name] [file] [version]",
```

Comment:
How can we indicate that version is optional?  I'm used to seeing square brackets for optional stuff, but everything is already in square brackets here...